docs/internal_api.yml

```diff
@@ -6,9 +6,9 @@ info:
     An API for user administration and user authentication handling. Not exposed via the API Gateway - intended for internal use only.
 
 basePath: '/api/internal/v1/useradm'
-host: 'mender-device-auth:8080'
+host: 'hosted.mender.io'
```

Comment:
This API is not exposed on hosted mender, and it is considered internal.
I don'w know what the policy is for these end-points, but hosted.mender.io is wrong.

Reply:
Thanks. Fixed now.